rollup.config.js

```diff
@@ -17,6 +17,7 @@ export default {
 	client: {
 		input: config.client.input(),
 		output: config.client.output(),
+		cache: true,
```

Comment:
I was under the impression `true` is the default. Looking at the rollup types, `true` is not even a valid option. Are you actually seeing this make a difference to dev mode recompile times?

- https://github.com/rollup/rollup/blob/master/src/rollup/types.d.ts#L486
- https://rollupjs.org/guide/en/#cache

Reply:
i changed to optional config

Reply:
@mehdiraized the comment you have in the code is invalid. It says to change to `true`, but per @MaxMilton's earlier comment and the Rollup documentation he shared `true` is not a valid option. It also doesn't seem like there's value to having a default value of `false` in the template